data/coalition missions.txt

```diff
@@ -60,12 +60,19 @@ mission "Coalition: First Contact"
 			label oppressive
 			`	The spider says, "In separate boxes, they each species keep. And in subtle ways, your development and discoveries they limit. And on your technology, restrictions they place."`
 			`	The beetle says, "Or, wrong are we? And the jump drive that brought you here, a gift from the Quarg it is? They its secret no longer hoard?" You tell it that the Quarg do indeed refuse to give jump drives to humans, and it says, "Then, prisoners indeed you are."`
-			label next
+			branch next
```

Comment:
The `goto next` on line 59 now leads line 71 so if you select "How did you defeat the Quarg?" you won't get the option to mention the Hai.